src/routes/(split)/components/theme-toggle-button/+page.md

```diff
@@ -0,0 +1,15 @@
+<script lang="ts">
+    import ThemeToggleButton from "$lib/components/ThemeToggleButton.svelte";
+</script>
+
+# Theme Toggle Button
+
+In addition to [theming](/styling/theming) we also provide an opinionated toggle to integrate easily a theme switcher in your dapp.
+
+```javascript
+<ThemeToggleButton />
+```
+
+## Showcase
+
+<ThemeToggleButton />
```

Comment:
Not important at all, but I was wondering if it makes sense to put the switch and the button on the same show-case page so you can see them side by side and know the other one also exists if you look at one.